src/containers/Tenant/utils/schemaActions.tsx

```diff
@@ -290,6 +290,7 @@ export const getActions =
             database: DB_SET,
 
             directory: DIR_SET,
+            resource_pool: DIR_SET,
```

Comment:
This includes create table / topic /  dir / topic options, I doubt that resource pool has all of it (you set it as childless). Probably, only copy options should be here

Reply:
Maybe it should be similar to index table `index_table: JUST_COPY,`